instrumentation/opentelemetry-instrumentation-urllib/tests/test_metrics_instrumentation.py

```diff
@@ -111,14 +141,196 @@ def test_basic_metric(self):
                 self.create_histogram_data_points(
                     result.length,
                     attributes={
-                        "http.status_code": str(result.code),
+                        "http.status_code": int(result.code),
                         "http.method": "GET",
                         "http.url": str(result.url),
                         "http.flavor": "1.1",
                     },
                 ),
             )
 
+    def test_basic_metric_new_semconv(self):
+        start_time = default_timer()
+        with request.urlopen(self.URL) as result:
+            duration_s = default_timer() - start_time
+
+            metrics = self.get_sorted_metrics()
+            self.assertEqual(len(metrics), 3)
+
+            (
+                client_request_body_size,
+                client_request_duration,
+                client_response_body_size,
+            ) = metrics[:3]
+
+            self.assertEqual(
+                client_request_duration.name, HTTP_CLIENT_REQUEST_DURATION
```

Comment:
Wdyt use the plain string as we did in urllib3?  I'm also fine if we leave to a follow-up PR